spec/unit/room-state.spec.ts

```diff
@@ -1122,59 +1122,4 @@ describe("RoomState", function () {
             ).toBeFalsy();
         });
     });
-    describe("skipWrongOrderRoomStateInserts", () => {
```

Comment:
Can we make this PR close:
https://github.com/matrix-org/matrix-js-sdk/pull/4532

when merged? Since it includes the revert changes?